app/javascript/components/item-review-card.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+
+type Props = {
```

Comment:
Esta información en verdad es toda la información que tiene una review o no? Derrepente se puede poner que la prop sea la review nomás.

Reply:
Sii, o sea en vez de userEmail es User en la info de una Review. Pero claro, la prop podría ser una Review.

Reply:
Floo, tengo una duda igual aquí. Primero, tengo el componente `item-review-list` en el cual se hace un v-for y a cada componente `item-review-card` se le entrega un review. 
Pero ahora con este cambio, de que la prop sea tipo Review como que quizás estoy encadenando mucho? onda, desde `item-details` se instancia `item-review-list` a quien se le pasan todas las reviews de un producto y luego dentro de este se hace el v-for para instanciar `item-review-card` por cada review. Quizás es medio code smell, middle man?